Upstream-Status: Backport

From ccf671f5b7fbbef53d9cfd3091229b0b2a25159d Mon Sep 17 00:00:00 2001
From: Edward Sheldrake <ejs1920@sourceforge.net>
Date: Wed, 13 Jun 2012 12:09:21 -0700
Subject: [PATCH] Fix --disable-canusb.

Reviewed-By: Guy Harris <guy@alum.mit.edu>

sgw - Removed configure part of the patch since it could change
Signed-off-by: Saul Wold <sgw@linux.intel.com>
---
 config.h.in  |    3 -
 configure    |10672 ++++++++++++++++++++++++++++++++++++++++------------------
 3 files changed, 7308 insertions(+), 3369 deletions(-)

diff --git a/config.h.in b/config.h.in
index bd5177c..aed5c37 100644
--- a/config.h.in
+++ b/config.h.in
@@ -232,9 +232,6 @@
 /* Define to the one symbol short name of this package. */
 #undef PACKAGE_TARNAME
 
-/* Define to the home page for this package. */
-#undef PACKAGE_URL
-
 /* Define to the version of this package. */
 #undef PACKAGE_VERSION
 
